test/log4j2.properties

```diff
@@ -0,0 +1,19 @@
+status = error
+dest = err
+name = PropertiesConfig
+
+filter.threshold.type = ThresholdFilter
+filter.threshold.level = debug
+
+appender.console.type = Console
+appender.console.name = STDOUT
+appender.console.layout.type = PatternLayout
+appender.console.layout.pattern = %d | %-5p | [%t] | %c | %m%n
+
+rootLogger.level = info
+rootLogger.appenderRef.stdout.ref = STDOUT
+
+# Set this to debug to log all data to/from server
+# See https://hc.apache.org/httpcomponents-client-4.5.x/logging.html
+logger.wire.name = org.apache.http.wire
+logger.wire.level = info
```

Comment:
Changes from the original logger:

- Removed the file appender (I can add it back if desired)
- Removed the `ES` and `HTTP` from layout pattern